ec2-backup.sh

```diff
@@ -21,17 +21,22 @@ mount_dir= ssh ec2-user@$publicDns 'sudo su file -s /dev/sdf | mkfs -t ext4 /dev
 ##
 ##FUNCTIONS
 ##
-method_type()
-{
-	if [$m = rysnc]
-		then
-			rsync -az $dir ec2-user@$publicDns:/dev/sdf
-	else
-		dd if=$dir of=$publicDns:/dev/sdf bs=$CHECK
-	fi	
+
+generateKeyPair() {
+
+        aws ec2 create-key-pair --key-name ec2BackUpKeyPair --output text > ~/ec2BackUpKeyPair.pem | chmod 600 ~/ec2BackUpKeyPair.pem
+}
+
+runInstance() {
+
+	aws ec2 run-instances --instance-type t1.micro --key ec2BackUpKeyPair --image-id ami-c27e48aa
+	publicDns = aws ec2 describe-instances | grep PublicDns | head -1 | awk '{print $2}' | sed 's/\"//g' | sed 's/\,//g'
```

Comment:
@rfarwaha  I removed the file you were creating locally as we cannot have anything generated locally based on instructions.